translation_of_critical_covid_data_visualizations_paper.Rmd

```diff
@@ -0,0 +1,8 @@
+#Translation of critical covid data visualizations paper
```

Comment:
Hi Siqi and Yaoling, please add a single whitespace between hashtag and your title.